core/pom.xml

```diff
@@ -138,22 +138,6 @@
   </dependencies>
 
   <build>
-    <resources>
```

Comment:
It was dead code right? Maybe should have been in a separate commit.

Reply:
Yep, split in 2 commits